libs/sdk-backend-bear/src/backend/workspace/metadata/index.ts

```diff
@@ -220,4 +224,70 @@ export class BearWorkspaceMetadata implements IWorkspaceMetadata {
             }),
         );
     }
+
+    public getUriIdentifierObjects = async (identifiers: Identifier[]): Promise<IUriIdentifierObject[]> => {
+        const uriIdentifierPairs = await this.authCall((sdk) =>
+            sdk.md.getUrisFromIdentifiers(this.workspace, identifiers),
+        );
+        return uriIdentifierPairs.map(convertUriIdentifierPair);
+    };
+
+    public getAttributeDisplayFormsByUris = async (
+        uris: Uri[],
+    ): Promise<IAttributeDisplayFormMetadataObject[]> => {
+        const wrappedAttributeOrDisplayForms = await this.authCall((sdk) =>
+            sdk.md.getObjects<GdcMetadata.IWrappedAttributeDisplayForm>(this.workspace, uris),
+        );
+        return wrappedAttributeOrDisplayForms.map((wrapperDisplayForm) => {
+            const displayFormDetails = wrapperDisplayForm.attributeDisplayForm;
+            const ref: UriRef = uriRef(displayFormDetails.meta.uri);
+            const attrRef = uriRef(displayFormDetails.content.formOf);
+
+            return newAttributeDisplayFormMetadataObject(ref, (df) =>
+                df
+                    .attribute(attrRef)
+                    .title(displayFormDetails.meta.title)
+                    .description(displayFormDetails.meta.summary)
+                    .default(displayFormDetails.content.default)
+                    .id(displayFormDetails.meta.identifier)
+                    .uri(displayFormDetails.meta.uri),
+            );
+        });
+    };
+
+    public getAttributesByUris = async (uris: Uri[]): Promise<IAttributeMetadataObject[]> => {
+        const wrappedAttributes = await this.authCall((sdk) =>
+            sdk.md.getObjects<GdcMetadata.IWrappedAttribute>(this.workspace, uris),
+        );
+
+        return wrappedAttributes.map((wrappedAttribute) => {
+            const {
+                meta: { title, uri, isProduction, identifier, summary },
+                content: { displayForms },
+            } = wrappedAttribute.attribute;
+            const refDisplayForm = uriRef(displayForms[0].content.formOf);
+            const attributeDisplayForms = displayForms.map((displayForm) =>
+                newAttributeDisplayFormMetadataObject(uriRef(displayForm.meta.uri), (df) =>
+                    df
+                        .attribute(refDisplayForm)
+                        .title(displayForm.meta.title)
+                        .description(displayForm.meta.summary)
+                        .default(displayForm.content.default)
+                        .id(displayForm.meta.identifier)
+                        .uri(displayForm.meta.uri)
+                        .displayFormType(displayForm.content.type),
+                ),
+            );
```

Comment:
this and the code in getAttributeDisplayFormsByUris look similar

Reply:
fixed